Sources/FluentUI_iOS/Components/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -837,7 +837,9 @@ open class BottomCommandingController: UIViewController, TokenizedControl {
         let itemImage = item.image ?? UIImage()
         let itemTitle = item.title ?? ""
         let tabItem = TabBarItem(title: itemTitle, image: itemImage, selectedImage: item.selectedImage, largeContentImage: item.largeImage)
-        let itemView = TabBarItemView(item: tabItem, showsTitle: itemTitle != "")
+        let itemView = TabBarItemView(item: tabItem,
+                                      showsTitle: itemTitle != "",
+                                      usesGlassEffectColors: bottomSheetControllerStyle == .glass)
```

Comment:
before 34 releases and this becomes permanent, can we rename bottomSheetControllerStyle to style

Reply:
renamed bottomSheetControllerStyle to style within the scope of BottomSheetController. I think it makes sense to retain bottomSheetControllerStyle in BottomCommandingController because currently, this Commanding control can be a sheet or Bar. Same philosophy as using .style within a Button's init VS buttonStyle in the external areas.

Reply:
Yep! the glass has only been defined for the BottomSheet Control. BottomBar is like this floating bar, probably for iPad? But yeah that won't get the glass treatment.